hikari/api/chunker.py

```diff
@@ -38,7 +91,44 @@ class GuildChunker(abc.ABC):
     __slots__: typing.Sequence[str] = ()
 
     @abc.abstractmethod
-    async def request_guild_chunk(self, guild: guilds.GatewayGuild) -> None:
+    def fetch_members_for_guild(
+        self,
+        guild: snowflakes.SnowflakeishOr[guilds.GatewayGuild],
+        *,
+        timeout: int,
+        include_presences: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
+        limit: int = 0,
+        query: str = "",
+        users: undefined.UndefinedOr[typing.Sequence[snowflakes.SnowflakeishOr[users_.User]]] = undefined.UNDEFINED,
+    ) -> iterators.LazyIterator[shard_events.MemberChunkEvent]:
+        ...
+
+    @abc.abstractmethod
+    async def get_chunk_status(self, nonce: str) -> typing.Optional[ChunkInformation]:
+        ...
+
+    @abc.abstractmethod
+    async def list_chunk_statuses_for_shard(self, shard_id: int) -> typing.Sequence[ChunkInformation]:
+        ...
+
+    @abc.abstractmethod
+    async def list_chunk_statuses_for_guild(self, guild_id: snowflakes.Snowflake) -> typing.Sequence[ChunkInformation]:
+        ...
```

Comment:
that's on the chunker itself, not the chunk info object and while intents only allow one guild per chunk request, chunk requests can be queued and this will ultimately be tracking multiple events at a time as that's its main purpose